src/msg_handler.c

```diff
@@ -285,6 +310,57 @@ void handle_msg(char* buf, char* response, int max_len) {
                 append_str(response, "NACK:Unknown coefficient command", max_len);
                 return;
             }
+        } else if (strcmp(next_arg, "BRAIN") == 0) {
+            next_arg = get_next_arg(response, "NACK:Missing brain command", max_len);
+            if(next_arg == NULL) {return;}
+            if (strcmp(next_arg, "SET") == 0) {
+                next_arg = get_next_arg(response, "NACK:Missing brain enable argument", max_len);
+                if(next_arg == NULL) {return;}
+
+                // Enable output
+                if (next_arg[0] == '1') {
+                    enable_output(BRAIN_OUTPUT, true);
+
+                    append_str(response, "ACK", max_len);
+                    return;
+                } else if (next_arg[0] == '0') {
+                    enable_output(BRAIN_OUTPUT, false);
+
+                    append_str(response, "ACK", max_len);
+                    return;
+                }
+
+                append_str(response, "NACK:Invalid brain enable argument", max_len);
+                return;
+            } else {
+                append_str(response, "NACK:Unknown brain command", max_len);
+                return;
+            }
+        } else if (strcmp(next_arg, "FAN") == 0) {
```

Comment:
Issue: `FAN` is currently an undocumented command.

Reply:
Fixed in edaed6838e8d4d457ca0d050f2e1e544a33b9ba0